definitions/npm/semver_v7.x.x/flow_v0.104.x-/semver_v7.x.x.js

```diff
@@ -0,0 +1,689 @@
+declare module "semver" {
+  import type semverParse from './functions/parse';
+  import type semverValid from './functions/valid';
+  import type semverClean from './functions/clean';
+  import type semverInc from './functions/inc';
+  import type semverDiff from './functions/diff';
+  import type semverMajor from './functions/major';
+  import type semverMinor from './functions/minor';
+  import type semverPatch from './functions/patch';
+  import type semverPrerelease from './functions/prerelease';
+  import type semverCompare from './functions/compare';
+  import type semverRcompare from './functions/rcompare';
+  import type semverCompareLoose from './functions/compare-loose';
+  import type semverCompareBuild from './functions/compare-build';
+  import type semverSort from './functions/sort';
+  import type semverRsort from './functions/rsort';
```

Comment:
All those does not work. 
* The files doesn't exist for a start...
* You can't import from other files in libdefs
* Types must start with a cap (convention)
* Named exports needs wrapping curlies

Reply:
Ok, I understand a bit better what your trying to achieve.
It should be more in this vein:

```js
declare module "semver" {
  import typeof { foo } from 'semver/foo';
}

declare module "semver/foo" {
  declare export function foo();
}
```